ext/random/php_random_module.h

```diff
@@ -0,0 +1,67 @@
+/*
+   +----------------------------------------------------------------------+
+   | Copyright (c) The PHP Group                                          |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | https://www.php.net/license/3_01.txt                                 |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Author: Go Kudo <zeriyoshi@gmail.com>                                |
+   +----------------------------------------------------------------------+
+ */
+
+#ifndef PHP_RANDOM_MODULE_H
+#define PHP_RANDOM_MODULE_H
+
+extern zend_module_entry php_random_module_entry;
+#define phpext_random_ptr &php_random_module_entry
+
+extern PHPAPI zend_class_entry *php_random_ce_Random_NumberGenerator_RandomNumberGenerator;
```

Comment:
nit: I think `random_ce_` would be shorter and follow conventions, e.g. spl starts with spl.

The other names in this header are fine.

```
extern PHPAPI zend_class_entry *spl_ce_SplFixedArray;
```